Documentation/docker/Dockerfile

```diff
@@ -1,5 +1,13 @@
 FROM swaggerapi/swagger-ui:v4.12.0
 
+# Definer bruger- og gruppe-id
+ARG USER_ID=1000
+ARG GROUP_ID=1000
```

Comment:
Nogen mener at best practice at anvende UID og GID >=10.000. Problemet er at, hvis det det lykkes for "the bad guys" og slippe ud af containeren, så er der større risiko for at 1.000 koliderer med en normal bruger der har nogle friske sudo rettigheder og så er der virkelig ballade. Så jeg synes vi skal gå efter => 10.000.